.github/actions/go-test/entrypoint.sh

```diff
@@ -34,9 +34,7 @@ if [ -z "$INPUT_DIR" ]; then
 fi
 
 # Need to fetch golang.org/x/* dependencies
-if ! [ -d "${GITHUB_WORKSPACE}/vendor/golang.org" ]; then
-	go mod vendor
-fi
+go mod vendor -v
```

Comment:
That condition is there in case someone wants to test `.github/actions/go-test/entrypoint.sh` locally without getting their modules wiped out and replaced each time.

What we should really be doing is using the [`actions/setup-go`](https://github.com/actions/setup-go#caching-dependency-files-and-build-outputs) action with `cache: true` to install Go, which would handle dependency caching for us (and would run `go mod vendor` itself.